.circleci/config.yml

```diff
@@ -49,12 +49,12 @@ jobs:
   e2e:
     environment:
       - CHANGE_MINIKUBE_NONE_USER=true
-      - K8S_VERSION=v1.13.2
+      - K8S_VERSION=v1.15.0
```

Comment:
In the future, we may want to target our current latest-stable-kops version of k8s.